docs/user/reference/metrics/text.md

```diff
@@ -2,6 +2,13 @@
 
 Records a single long Unicode text, used when the limits on `String` are too low.
 
+{{#include ../../shared/blockquote-stop.html}}
```

Comment:
This blockquote type is a _very_ strong visual warning (example: https://mozilla.github.io/glean/book/appendix/contribution-guidelines.html#stop-quote). From the bug it seems like this issue is under control i.e. we have a plan on how to deal with it. I wonder if it's even necessary to add this warning here at all. Do you know a rough timeline for the issue to be fixed and for this warning to be stale? 

If we keep it, let's please rephrase it and also not use such a strong visual cue :) I worry that a warning phrased and presented like this could cause a long lasting trust issue related to these metric types or even to Glean itself. We have had such problems with less strong warnings on the book e.g. https://bugzilla.mozilla.org/show_bug.cgi?id=1734454.